guides/micronaut-mongodb-synchronous/java/src/main/java/example/micronaut/Fruit.java

```diff
@@ -0,0 +1,47 @@
+package example.micronaut;
+
+import io.micronaut.core.annotation.Creator;
+import io.micronaut.core.annotation.Introspected;
+import io.micronaut.core.annotation.NonNull;
+import io.micronaut.core.annotation.Nullable;
+
+import javax.validation.constraints.NotBlank;
+
+@Introspected // <1>
+public class Fruit {
+
+    @NonNull
+    @NotBlank // <2>
+    private String name;
```

Comment:
if possible, make this final and remove the setter

Reply:
not possible because of https://github.com/micronaut-projects/micronaut-guides/pull/528#discussion_r709156589